.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,20 @@ This pull request closes issue # (_put the issue number here_)
 - [ ] All code is written in untranspiled ECMAScript (ES 2017) and is formatted using [prettier](https://prettier.io)
 - [ ] Necessary unit tests are added in order to ensure correct behavior
 - [ ] Documentation has been added
+
+<details>
```

Comment:
Could you elaborate a little on what those commands are & what ppl are supposed to do with them?

Reply:
@herschel666 I added a small introduction what these commands are used for. Does that help?
These commands are mainly for us, in case someone else uses them, bors will probably comment on the issue that they don't have the correct access rights.
Also when we delegate, bors will comment on the issue mentioning that the PR author (or another user) now has the rights to merge that PR.